### PR TITLE
remove .html from urls

### PR DESCRIPTION
**Pull request by Sreenivas-Kalluru**
remove .html from urls

### CODE REVIEW

**Comment by Sreenivas-Kalluru** [comment]
@ras0219-msft Could you please review and let me know if any comments

File: scripts/validateLinks.js [hunk 1]
```diff
@@ -45,7 +45,12 @@ async function load_page_info(page, relative_path) {
         if (match[1].length == 0) {
             if (match[3].length > 0) {
                 // reference to self anchor
-                ret.links.push([relative_path, match[3]]);
+                if (relative_path.endsWith(".html")) {
+                    ret.links.push([relative_path, match[3]]);
```
Comment (ras0219-msft) [inline scripts/validateLinks.js:49]: Should this case be turned into an error to ensure we have converted all `.html` links?

Comment (Sreenivas-Kalluru) [reply scripts/validateLinks.js:49]: this shouldn't cause any error since relative paths referred to href attribute values.

**Comment by dan-shaw** [comment]
Can you explain what this fixes?

**Review by dan-shaw — APPROVED** [review]

**Pull request merged**